Sources/NIOSSL/SSLCertificate.swift

```diff
@@ -102,7 +111,7 @@ public class NIOSSLCertificate {
     ///
     /// In general, however, this function should be avoided in favour of one of the convenience
     /// initializers, which ensure that the lifetime of the `X509` object is better-managed.
-    static public func fromUnsafePointer(takingOwnership pointer: UnsafeMutablePointer<X509>) -> NIOSSLCertificate {
+    static func fromUnsafePointer(takingOwnership pointer: UnsafeMutablePointer<X509>) -> NIOSSLCertificate {
```

Comment:
This is technically a breaking API change. Do we want to ship a 3.0 here? Or do we think this can be made an exception?

Reply:
Yup, my vote goes for exception. There was never a legal way to obtain an `UnsafeMutablePointer<X509>` (with the _right_ `X509`...).

Reply:
I think I agree.